### ISSUE TITLE
version 0.4.1 does not compile with swift 2.3

### ISSUE THREAD

**Issue by sherm4n90** [shogo4405/lf.swift#116]
@shogo4405  There is a few compiling errors in the version 0.4.1 when compiling using swift 2.3.
What's the solution? 
Thanks

**Comment by shogo4405** [comment]
are u using xcode8 or 7?
if u are using xcode8, no way. plese cosider downgrade to xcode7.3.

**Comment by sherm4n90** [comment]
@shogo4405  Using Xcode 8, I need to stay in swift 2.3 but with SDK for iOS10, so i cant downgrade to Xcode 7.3. It compiles now by doing a convert, which is not ideal, can I include the library manually so every time a make a pod install i don't have to do the convert process?

**Comment by alex-vasenin** [comment]
I'm using Xcode 8 and Swift 3. After adding pod 'lf' to the Podfile, Xcode asked me to convert both lf and XCGLogger, but then said no changes needed. I don't get it, why it asks to convert in the first place?

**Comment by alex-vasenin** [comment]
Here a workaround for the "convert after every pod install" issue found at https://github.com/CocoaPods/CocoaPods/issues/5521

```
post_install do |installer|
  installer.pods_project.targets.each do |target|
    target.build_configurations.each do |config|
      config.build_settings['SWIFT_VERSION'] = '3.0'
    end
  end
end
```

**Comment by shogo4405** [comment]
0.4.2 supports XCode8. Try update. I released 0.4.2.

```
platform :ios, ‘8.0’

target 'Test' do
  use_frameworks!
  pod 'lf', '~> 0.4.2'
end

post_install do |installer|
  installer.pods_project.targets.each do |target|
    target.build_configurations.each do |config|
      config.build_settings['SWIFT_VERSION'] = '2.3'
    end
  end
end

```
